src/site/content/en/index.md

```diff
@@ -1,25 +1,62 @@
 ---
-layout: homepage
-description: |
-  Get the web's modern capabilities on your own sites and apps with useful
-  guidance and analysis from web.dev.
-date: 2018-11-05
-i18n:
-  headline: Let's build the future of the web
-  subheadline: |
-    Get the web's modern capabilities on your own sites and apps with useful
-    guidance and analysis from web.dev.
-  test_cta: Test my site
-  about_subheadline: |
-    As the web advances, users' expectations grow. With web.dev's guidance, you
-    can give your users the best experience, wherever they are.
-  about_cta: About web.dev
-  explore_cta: Explore topics
-  latest_headline: Latest stories
-  latest_subheadline: See our latest news, updates, and stories for developers.
-  see_all: See all articles
-  explore_subheadline: |
-    Learn more about a topic, and practice your skills with a hands-on
-    codelab.
-  see_topics: See all topics
+layout: 'homepage-next'
+title: 'web.dev'
+intro:
+  eyebrow: 'About web.dev'
+  title: "Let's build the future of the web, together"
+  summary: |
+    'Take advantage of the latest modern technologies to build amazing web experiences for everyone'
+  buttonText: 'Learn more'
+  buttonUrl: '/about/'
+  image: 'https://web-dev.imgix.net/image/VbAJIREinuYvovrBzzvEyZOpw5w1/B9KWrQo39fXbKSVYjz1e.svg'
+  imageWidth: '330'
+  imageHeight: '210'
+featureCard:
+  theme: 'dark'
+  title: 'Explore structured learning paths to learn new skills'
+  eyebrow: 'Learn'
+  url: '/learn'
+  background: 'https://web-dev.imgix.net/image/jxu1OdD7LKOGIDU7jURMpSH2lyK2/yJSJQfrrK4pVh44yuMZV.svg'
+promoPanel:
+  title: 'Chrome Dev Summit 2021'
+  summary: 'Join the Chrome team for workshops, office hours, and learning lounges taking place throughout November.'
+  primaryButtonUrl: 'https://events.withgoogle.com/chrome-dev-summit-2021/registrations/new/details/'
+  primaryButtonText: 'REQUEST AN INVITE'
```

Comment:
we should probably make this text all-caps using `text-transform` rather than have the string in caps. I think some screen readers announce all caps text differently.

Reply:
Fixed